playground/frontend/lib/modules/examples/components/example_list/expansion_panel_item.dart

```diff
@@ -41,19 +41,21 @@ class ExpansionPanelItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    AnalyticsService analyticsService = AnalyticsService.get(context);
+
     return Consumer2<PlaygroundState, ExampleState>(
       builder: (context, playgroundState, exampleState, child) => MouseRegion(
         cursor: SystemMouseCursors.click,
         child: GestureDetector(
           onTap: () async {
             if (playgroundState.selectedExample != example) {
               closeDropdown(exampleState);
-              AnalyticsService.get(context).trackSelectExample(example);
               final exampleWithInfo = await exampleState.loadExampleInfo(
                 example,
                 playgroundState.sdk,
               );
               playgroundState.setExample(exampleWithInfo);
+              analyticsService.trackSelectExample(exampleWithInfo);
```

Comment:
What is the benefit of this?

Reply:
I think that this method allows us to sort examples in our bucket (to delete unused f.e.).

Reply:
This method logs the event of changing the example to analytics. I mean what is the benefit of:
1. Calling it after `setExample`?
2. Calling it with `exampleWithInfo`?

I can see one downside: Later analytics logging means lost events in edge cases.
Also this seems unrelated to the issue of this PR.